docs/standards/universal-profile/lsp6-key-manager.md

```diff
@@ -397,15 +197,181 @@ permissions: CHANGEPERMISSIONS + SETDATA
 
 </Tabs>
 
+</details>
+
 :::tip
 
-You can easily [`encodePermissions`](../../../../tools/erc725js/classes/ERC725#encodepermissions) and [`decodePermissions`](../../../../tools/erc725js/classes/ERC725#decodepermissions) with [**erc725.js**](../../../../tools/erc725js/getting-started).
+You can use the [`encodePermissions(...)`](../../../../tools/erc725js/classes/ERC725#encodepermissions) and [`decodePermissions(...)`](../../../../tools/erc725js/classes/ERC725#decodepermissions) functions from the [**erc725.js**](../../../../tools/erc725js/getting-started) tool to easily combine and decode LSP6 permissions.
 
 :::
 
-:::note
+---
 
-When deployed with our [**lsp-factory.js** tool](https://docs.lukso.tech/tools/lsp-factoryjs/introduction/getting-started/), the Universal Profile owner will have all the permissions above set by default.
+## Details about permission types
+
+:::caution
+
+To **add / remove entries in the list of allowed addresses, functions, standards or ERC725Y keys**, the **whole array** should be ABI-encoded again and reset. Each update **overrides the entire previous state**.
+
+Note that this process is expensive since the data being set is an ABI-encoded array.
+
+:::
+
+### Address Permissions
+
+An address can hold one (or more) permissions, enabling it to perform multiple _"actions"_ on an ERC725Account. Such _"actions"_ include **setting data** on the ERC725Account, **calling other contracts**, **transferring native tokens**, etc.
+
+To grant permission(s) to an `<address>`, set the following key-value pair below in the [ERC725Y](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-725.md#erc725y) contract storage.
+
+- **key:** `0x4b80742d0000000082ac0000<address>`
+- **value:** one of the available options below.
+
+```json
+{
+  "name": "AddressPermissions:Permissions:<address>",
+  "key": "0x4b80742d0000000082ac0000<address>",
```

Comment:
it is maybe necessary to write that address should be 40 chars long - without the `0x` prefix?